Puppeteer issue installing cli

Problem still persists with initial patch.

Error happens when installilng `igc` plugins for `oc`:

```
curl -sL https://cli.cloudnativetoolkit.dev/install.sh | sh
Installing version v1.43.1 of Cloud-Native Toolkit cli for macos-x64 into /usr/local/bin
######################################################################## 100.0%
Installing igc cli as plugins to kubectl/oc clis
pkg/prelude/bootstrap.js:1876
      throw error;
      ^

Error: Cannot find module 'puppeteer'
Require stack:
- /snapshot/ibm-garage-cloud-cli/dist/services/generate-token/generate-token-impl.js
- /snapshot/ibm-garage-cloud-cli/dist/services/generate-token/index.js
- /snapshot/ibm-garage-cloud-cli/dist/services/jenkins-auth/config-jenkins-auth.js
- /snapshot/ibm-garage-cloud-cli/dist/services/jenkins-auth/index.js
- /snapshot/ibm-garage-cloud-cli/dist/services/index.js
- /snapshot/ibm-garage-cloud-cli/dist/commands/console.js
- /snapshot/ibm-garage-cloud-cli/node_modules/yargs/index.cjs
- /snapshot/ibm-garage-cloud-cli/dist/script.js
1) If you want to compile the package/file into executable, please pay attention to compilation warnings and specify a literal in 'require' call. 2) If you don't want to compile the package/file into executable and want to 'require' it from filesystem (likely plugin), specify an absolute path in 'require' call using process.cwd() or process.execPath.
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function._resolveFilename (pkg/prelude/bootstrap.js:1955:46)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at Module.require (pkg/prelude/bootstrap.js:1855:31)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/snapshot/ibm-garage-cloud-cli/dist/services/generate-token/generate-token-impl.js:6:21)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Module._compile (pkg/prelude/bootstrap.js:1937:32)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/snapshot/ibm-garage-cloud-cli/dist/services/generate-token/generate-token-impl.js',
    '/snapshot/ibm-garage-cloud-cli/dist/services/generate-token/index.js',
    '/snapshot/ibm-garage-cloud-cli/dist/services/jenkins-auth/config-jenkins-auth.js',
    '/snapshot/ibm-garage-cloud-cli/dist/services/jenkins-auth/index.js',
    '/snapshot/ibm-garage-cloud-cli/dist/services/index.js',
    '/snapshot/ibm-garage-cloud-cli/dist/commands/console.js',
    '/snapshot/ibm-garage-cloud-cli/node_modules/yargs/index.cjs',
    '/snapshot/ibm-garage-cloud-cli/dist/script.js'
  ],
  pkg: true
}
```

--- a/src/services/index.ts
+++ b/src/services/index.ts
@@ -10,7 +10,6 @@ export * from './endpoints';
 export * from './git-secret';
 export * from './gitops-init';
 export * from './gitops-module';
-export * from './jenkins-auth';
 export * from './login';
 export * from './namespace';
 export * from './register-pipeline';

